meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx_2022.01.bb

```diff
@@ -3,7 +3,7 @@ UBOOT_VERSION = "v2022.01"
 UBRANCH = "xilinx-v2022.01-rebase"
 UBOOTURI = "git://github.com/foundriesio/u-boot.git;protocol=https"
 
-SRCREV = "a056b687f497022ad2aeaf40ef44d550448c31ff"
+SRCREV = "49aea78e3033e9d78108f7d12cc98ba701632f4a"
```

Comment:
We should probably bump to the latest now, including your latest pr that is now approved.